com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.RenderGraphUtils.cs

```diff
@@ -68,6 +68,15 @@ Color GetColorBufferClearColor(HDCamera hdCamera)
             if (CoreUtils.IsSceneFilteringEnabled())
                 clearColor.a = 0.0f;
 
+            // Get the background color from preferences if preview camera
+#if UNITY_EDITOR
+            if (HDUtils.IsRegularPreviewCamera(hdCamera.camera) && hdCamera.camera.clearFlags != CameraClearFlags.SolidColor)
```

Comment:
The idea is that if the user wants to clear in some particular way, they set the clear flags. So if they're setting that to `CameraClearFlags.SolidColor`, that's a very clear indication that we should clear using the color they set in `Camera.backgroundColor`. Otherwise we fall back to the default background color.

Reply:
All right I see, thanks.